### ISSUE TITLE
Using coconut instead?

### ISSUE THREAD

**Issue by janfreyberg** [rb643/resting_state_eeg#3]
One idea I had (and something I've been wanting to try for a while) is using coconut instead of normal python. Coconut is a kind of "extended" python with some extra features. So all code that runs in python will run in coconut as well, but it also has pipe operators (like `%>%` in R and dplyr):

```python
4 |> maths.sqrt() |> print
```

is equivalent to:

```python
print(maths.sqrt(4))
```

Since we're making a *pipeline*, having a _pipe operator_ could be cool 😃

**Comment by janfreyberg** [comment]
more info here:
http://coconut-lang.org/

**Comment by rb643** [comment]
Yeah go wild :) I'm off jupyter now so you can edit away on the notebook as well if you want!
